src/ert/parsing/lark_parser.py

```diff
@@ -362,7 +362,8 @@ def _handle_includes(
                         filename=config_file,
                     ).set_context(error_context)
                 )
-                continue
+
+                args = args[0:1]
```

Comment:
this seems a bit unexpected / out of place. what is happening here, and how is it related to the issue this PR fixes? or does this fall under camp-site rule?

Reply:
On a second look, this change makes sense but is not directly related to this PR, just a very tiny thing I noticed doing this, so I guess it does fall under the camp-site rule.

If an include has multiple arguments, it will now keep only the first, and (1) lint that the rest are superfluous, and (2) potentially also lint errors pertaining to the actually included file. With the continue in place, it will only do (1) because the continue stops the execution.

Reply:
put it into its own commit, write a test if you can, and then you will have earned a(nother) gold star from me : ) (and can merge it in this PR, i think)